app/endpoints/calendar.py

```diff
@@ -228,6 +228,25 @@ async def delete_bookings_id(
         )
 
 
+@router.post(
+    "/calendar/ical",
```

Comment:
I would suggest a route like `POST /calendar/ical/recreate` or `POST /calendar/ical/create` to emphasize what the endpoint is doing.

Reply:
/calendar/ical/create